src/data/actions/authentication.js

```diff
@@ -31,17 +30,8 @@ const fetchLoginFailure = error => ({
 const login = (email, password) => (
   (dispatch) => {
     dispatch(fetchLoginRequest());
-    return LmsApiService.authenticate(email, password)
+    return AuthService.login(email, password)
```

Comment:
Can you explain what when wrong with the original plan for the single login page?  Is this what made us have to use cross domain csrf?  I think it might expand the concerned pages when it comes to click-jacking as well.  I know you will be discussing with Nimisha as well when she returns.